src/routes/v2/authenticated/sites.ts

```diff
@@ -188,6 +196,23 @@ export class SitesRouter {
     return res.status(404).json({ message: "Unable to get staging status" })
   }
 
+  getLinkCheckerStatus: RequestHandler<
+    { siteName: string },
+    RepoErrorDto | ResponseErrorBody,
+    never,
+    never,
+    { userWithSiteSessionData: UserWithSiteSessionData }
+  > = async (req, res) => {
+    const { userWithSiteSessionData } = res.locals
+    const result = await this.repoCheckerService.checkRepo(
+      userWithSiteSessionData.siteName
+    )
+    if (result.isOk()) {
+      return res.status(200).json(result.value)
+    }
+    return res.status(400).json({ status: "error" })
```

Comment:
like the error toast ah?

Reply:
err not the error toast per se, but more the error message! (which the toast uses to display)

it's ok for now because we're testing this with users

Reply:
@kishore03109 are we updating the status code/error msg here?